src/legacy/ui/public/agg_types/buckets/migrate_include_exclude_format.d.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { AggConfig } from 'ui/vis/agg_config';
+
+export const isType: (agg: AggConfig) => any;
```

Comment:
This type definition doesn't match the function:

```javascript
function isType(type) {
  return function (agg) {
    const field = agg.params.field;
    return field && field.type === type;
  };
}
```

The type should be something like `export function isType(type: string): (agg: AggConfig) => any;`

Reply:
I think we can also specify the return type as boolean:

```
export function isType(type: string): (agg: AggConfig) => boolean;
```

Reply:
@wylieconlon @maryia-lapata good point! Thanks! Done